pyproject.toml

```diff
@@ -11,13 +11,21 @@ loguru = "^0.7.2"
 pytest = "^8.3.2"
 black = "^24.4.2"
 matplotlib = "^3.9.1"
+cytoolz = "^0.12.3"
```

Comment:
remove (because client don't like it :((((((

Reply:
rip done